src/app/core/api/logs.service.ts

```diff
@@ -0,0 +1,105 @@
+import { Injectable } from '@angular/core';
+import { CustomModelService } from '../utils/custom-model.service';
+import { LanguageService } from '../language/language.service';
+import { HttpService } from '../network/http.service';
+import { Log } from 'src/app/models/log';
+import { URL_BMS_API } from '../../../environments/environment';
+import { AppInjector } from 'src/app/app-injector';
+import { ProjectService } from './project.service';
+import { DistributionService } from './distribution.service';
+import { BeneficiariesService } from './beneficiaries.service';
+import { VendorsService } from './vendors.service';
+import { forkJoin } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LogsService extends CustomModelService {
+    customModelPath = 'logs';
+    readonly api = URL_BMS_API;
+
+    constructor(
+        protected http: HttpService,
+        protected languageService: LanguageService
+    ) {
+        super(http, languageService);
+    }
+
+    public fillWithOptions(log: Log) {}
```

Comment:
You do, because it is used in the modal service for opening the details of an object. Maybe we should change this as we are using the same empty function in 8 different services.

Reply:
Can't we just put the empty one at the parent level?

Reply:
Of course, shall I do it in the same branch or make a new PR?

Reply:
Same branch